modules/runner/src/integrationTest/java/org/apache/ignite/internal/sql/engine/ItAggregatesTest.java

```diff
@@ -64,6 +64,18 @@ public void countOfNonNumericField() {
                 .returns(15d, 1L)
                 .check();
 
+        // same query, but grouping by alias
+        assertQuery("select salary as sal, count(name) from person group by sal order by salary")
```

Comment:
Can we also add a test for computed column grouping?

```sql
select (salary / 4 + 10) as sal, count(*) from person group by sal order by sal
```

Reply:
Why not.

Have added such tests, but with a bit different expression. Still expression, though